src/components/TodosList/TodosList.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
`todoList`. Singular form should be used for things like "list, table, grid", etc.

Reply:
The `todosList `component has been renamed to the `todoList`